x-pack/plugins/alerting/server/task_runner/task_runner.test.ts

```diff
@@ -1573,16 +1547,33 @@ describe('Task Runner', () => {
           },
         ],
       });
+
+      alertsService.createAlertsClient.mockImplementation(() => {
```

Comment:
can we use the mock alerts client from alerts_client.mock.ts here?

Reply:
done